### PR TITLE
chore: update local node

### PR DESCRIPTION
**Pull request by georgi-l95**
**Description**:
This PR updates local-node to latest version and bumps all used images as well.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by AlfredoG87 — DISMISSED** [review]
Nice, do you know if this new versions fix the `nonce out of sync` issues we were experiencing?

File: packages/server/tests/helpers/prerequisite.ts [hunk 1]
```diff
@@ -11,19 +11,16 @@ const RELAY_URL = process.env.E2E_RELAY_HOST || LOCAL_RELAY_URL;
 
 (function () {
   if (USE_LOCAL_NODE) {
-    process.env['NETWORK_NODE_IMAGE_TAG'] = '0.45.0-alpha.0';
-    process.env['HAVEGED_IMAGE_TAG'] = '0.45.0-alpha.0';
-    process.env['MIRROR_IMAGE_TAG'] = '0.92.0';
+    const networkNodeImage = '0.47.0-alpha.4';
```
Comment (Nana-EC) [inline packages/server/tests/helpers/prerequisite.ts:14]: Q: why no longer process env variables.
Doing it as is means we can't override without code changes

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Question about hard coding

**Comment by georgi-l95** [comment]
@AlfredoG87 Yep, this new version have the needed changes.

File: packages/server/tests/helpers/prerequisite.ts (hunk 1 above)
Comment (georgi-l95) [reply packages/server/tests/helpers/prerequisite.ts]: Removed them, local node will be up to date with the latest images always, so no need here to do anything.

File: packages/server/tests/helpers/prerequisite.ts [hunk 2]
```diff
@@ -11,19 +11,11 @@ const RELAY_URL = process.env.E2E_RELAY_HOST || LOCAL_RELAY_URL;
...
 (function () {
   if (USE_LOCAL_NODE) {
-    process.env['NETWORK_NODE_IMAGE_TAG'] = '0.45.0-alpha.0';
-    process.env['HAVEGED_IMAGE_TAG'] = '0.45.0-alpha.0';
-    process.env['MIRROR_IMAGE_TAG'] = '0.92.0';
-
-    console.log(
-      `Docker container versions, services: ${process.env['NETWORK_NODE_IMAGE_TAG']}, mirror: ${process.env['MIRROR_IMAGE_TAG']}`,
-    );
-
```
Comment (AlfredoG87) [inline packages/server/tests/helpers/prerequisite.ts:21]: I think there are pros and cons of having the latest images always. we did have a regression lately due to that fact, the good thing is that it helped us to find issues faster and sooner, the bad is that it blocked the CI pipeline for a couple days.

@Nana-EC @ebadiere, What are your thoughts on this?

I personally prefer to get latests the sooner so we catch potencial issues earlier.

**Review by AlfredoG87 — COMMENTED** [review]
Leave a comment to start a discussion.

File: packages/server/tests/helpers/prerequisite.ts (hunk 2 above)
Comment (Nana-EC) [reply packages/server/tests/helpers/prerequisite.ts:21]: Same thoughts.
Latest is great but it takes control away from us.
If there's a way to have a back up config whereby if local node had a version different from what we want to test that would be good.
I think there's such a flow

Comment (georgi-l95) [reply packages/server/tests/helpers/prerequisite.ts:21]: If it's blocked we can just add wanted tag versions in the yml file, until issue is resolved.

Comment (Nana-EC) [reply packages/server/tests/helpers/prerequisite.ts:21]: I think let's get this in as noted.
@ebadiere is working on the Configurable Manual Tests that can help with concerns of running targeted tests on specific versions

**Review by AlfredoG87 — APPROVED** [review]

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**